test/api/__snapshots__/CLI.test.js.snap.webpack5

```diff
@@ -19,13 +19,13 @@ exports[`CLI API custom help output should display help information 1`] = `
 ",
   ],
   [
-    "[1mWebpack documentation:[22m https://webpack.js.org/.",
+    "Webpack documentation: https://webpack.js.org/.",
   ],
   [
-    "[1mCLI documentation:[22m https://webpack.js.org/api/cli/.",
+    "CLI documentation: https://webpack.js.org/api/cli/.",
   ],
   [
-    "[1mMade with ♥ by the webpack team[22m.",
+    "Made with ♥ by the webpack team.",
```

Comment:
@snitin315 Why it was changed?

Reply:
@alexander-akait Test case was updated to use `--mode` instead of `--color`.
https://github.com/webpack/webpack-cli/pull/4315#discussion_r1827175248